src/java/org/apache/cassandra/cql3/QueryProcessor.java

```diff
@@ -131,6 +128,16 @@ public class QueryProcessor implements QueryHandler
                     DatabaseDescriptor.getPreparedStatementsCacheSizeMiB());
     }
 
+    public static void evictPreparedStatement(MD5Digest key, RemovalCause cause)
```

Comment:
I would add annotation VisibleForTesting here? It is not intended to be called iiuc

Reply:
:+1:, looking at it again now I just realized this doesn't even need to be public, so I'll reduce the visibility to private. 

We ended up only needing this as a separate method to ensure it gets invoked in `PstmtPersistenceTest` and apparently byteman is still able to instrument around it with private visibility.